### PR TITLE
JBTM-3588 Jakarta Migration of the docs

### PR DESCRIPTION
**Pull request by mmusgrov**
https://issues.redhat.com/browse/JBTM-3588

The conversion was a bit tricky because there is a great deal of repetition in our docs, which would be nice to address sometime, and we make extensive use of the phrase JTA (I tried to define how we use early on in each document that references it.

### CODE REVIEW

File: project/en-US/jts/trailmap.xml [hunk 1]
```diff
@@ -1504,7 +1503,7 @@ TransactionManager.commit();
                 when the transaction terminates, e.g., are driven through the two-phase commit
                 protocol.
                 <para>
-                    The JTA is much more closely integrated with the XA concept of resources
+                    Jakarta Transactions is much more closely integrated with the XA concept of resources
```
Comment (marcosgopen) [inline project/en-US/jts/trailmap.xml:1506]: Jakarta Transactions **are** much more closely integrated with the XA concept of resources

**Review by marcosgopen — APPROVED** [review]
Thanks @mmusgrov for your PR.

**Comment by mmusgrov** [comment]
Thanks @marcosgopen Can you also check out my [PR](https://github.com/jbosstm/narayana.io/pull/61) containing the website updates.

**Comment by mmusgrov** [comment]
TESTIT

**Pull request merged**